### ISSUE TITLE
Unknown browser Electron

### ISSUE THREAD

**Issue by pop123123123** [ai/browserslist#192]
I use
```json
"browserslist": ["last 2 Electron versions"],
```
 in my package.json (browserslist 2.8), as advised in the README, but I get a 
```sh
BrowserslistError: Unknown browser Electron
```
If I replace it by Chrome versions, it works fine.

**Comment by pop123123123** [comment]
I just tried with
```json
"browserslist": ["last 2 iOS major versions"],
```
and I get a 
```sh
Unknown browser query `last 2 iOS major versions`
```
Do I open another issue ?

**Comment by ai** [comment]
@pop123123123 yeap, open different issue for it

**Comment by pop123123123** [comment]
I sent a PR, if you want to check it.

**Comment by ai** [comment]
Released in 2.9
